app/controllers/videos_controller.rb

```diff
@@ -15,4 +15,15 @@ def search
     @videos = Video.search_by_title(params[:search_string])
   end
 
+  def advanced_search
+    options = { reviews: params[:reviews] }
+    options[:rating_from] = params[:rating_from] if params[:rating_from].present?
+    options[:rating_to] = params[:rating_to] if params[:rating_to].present?
+
```

Comment:
i did it this way because i was seeing false failures for the "search with title & description" scenario.
'params[:rating_from]` & `params[:rating_to]` are set to an empty string `""` which screws up the options & were causing that failure.